kotlintest-assertions/src/commonMain/kotlin/io/kotlintest/properties/gens.kt

```diff
@@ -242,6 +242,17 @@ fun Gen.Companion.bool(): Gen<Boolean> = object : Gen<Boolean> {
    }
 }
 
+/**
+ * Returns a stream of randomly-chosen Chars
+ */
+fun Gen.Companion.char(): Gen<Char> = object : Gen<Char> {
+  override fun constants(): Iterable<Char> = emptyList()
+  override fun random(seed: Long?): Sequence<Char> {
+    val r = if (seed == null) Random.Default else Random(seed)
+    return generateSequence { r.nextPrintableChar() }
```

Comment:
I'm wondering if we want to limit ourselves to the "Printable Chars" only, or if we want to generate many other chars. What do you think?

Reply:
I think that is a good idea. I am working on another proposal.